releasenotes/notes/docker-image-secrets-script-efabab863b28ff5c.yaml

```diff
@@ -0,0 +1,2 @@
+features:
+  - Docker image: handle docker/kubernetes secret files with a helper script
```

Comment:
Pleas add quotes around the line. Since it contains a `:` it become a dict in yaml and reno will fail on it.

Or use the literal style:
```yaml
---
features:
  - |
    Docker image: handle docker/kubernetes secret files with a helper script
```